boards/arm/96b_neonkey/doc/96b_neonkey.rst

```diff
@@ -0,0 +1,194 @@
+********
+
+96Boards Neonkey board is based on the STMicroelectronics STM32F411CE
+Cortex M4 CPU. Zephyr applications use the 96b_neonkey configuration
+to run on these boards.
+
+.. figure:: img/96b-neonkey-front.jpg
+     :width: 500px
+     :align: center
+     :height: 330px
+     :alt: 96Boards Neonkey
+
+     96Boards Neonkey
+
+This board acts as a sensor hub platform for all 96Boards compliant
+family products. It can also be used as a standalone board.
+
+Hardware
+********
+
+96Boards Neonkey provides the following hardware components:
+
+- STM32F411CE in UFQFPN48 package
+- ARM |reg| 32-bit Cortex |reg|-M4 CPU with FPU
+- 100 MHz max CPU frequency
+- 1.8V work voltage
+- 512 KB Flash
+- 128 KB SRAM
+- On board sensors:
+
+  - Temperature/Humidity (SI7034-A10)
+  - Pressure (BMP280)
+  - ALS/Proximity (RPR-0521R5)
+  - Geomagnetic (BMM150)
+  - Acclerometer/Gyroscope (BMI160)
+  - Microphone (SPK0415HM4H-B)
+
+- 4 User LEDs
+- 15 General purpose LEDs
+- GPIO with external interrupt capability
+- I2C (3)
+- SPI (1)
+- I2S (1)
+
+Supported Features
+==================
+
+The Zephyr 96b_neonkey board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | system clock                        |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash                               |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+
+More details about the board can be found at `96Boards website`_.
+
+The default board configuration can be found in the defconfig file:
+
+        ``boards/arm/96b_neonkey/96b_neonkey_defconfig``
+
+Connections and IOs
+===================
+
+LED
+---
+
+- LED1 / User1 LED = PB12
+- LED2 / User2 LED = PB13
+- LED3 / User3 LED = PB14
+- LED4 / User4 LED = PB15
+
+Push buttons
+------------
+
+- BUTTON = RST (SW1)
+- BUTTON = USR (SW2)
+
+System Clock
+============
+
+96Boards Neonkey could be driven by internal oscillator as well as main PLL clock.
```

Comment:
change to;

     can be driven by an internal oscillator, as well as the main PLL clock.